src/pages/images/actions/ImportFromTarballBtn.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC } from "react";
+import { ActionButton } from "@canonical/react-components";
+import usePortal from "react-useportal";
+import ImportImageFromTarballForm from "./forms/ImportImageFromTarballForm";
+
+interface Props {
+  project: string;
+}
+
+const ImportFromTarballBtn: FC<Props> = ({ project }) => {
+  const { openPortal, closePortal, isOpen, Portal } = usePortal();
+
+  return (
+    <>
+      {isOpen && (
+        <Portal>
+          <ImportImageFromTarballForm close={closePortal} project={project} />
+        </Portal>
+      )}
+      <ActionButton className="u-no-margin--bottom" onClick={openPortal}>
+        <span>Import from Tarball</span>
```

Comment:
```suggestion
        <span>Upload image</span>
```
Maybe clearer?
Also this component could be UploadImageBtn and the form UploadImageForm, wdyt?

Reply:
Can do. I was following the naming convention of other buttons such as ExportImageTarballBtn, CreateInstanceFromImageBtn and UploadCustomIsoBtn. I can change this if you wish? I believe ImportFromTarballBtn is a bit more specific.

Reply:
CreateInstanceFromImageBtn is overly specific, because there is also another way of creating instances without selecting an image first.

Here I think we only have one way to upload/create an image, that's why I suggested the simpler UploadImage name. Maybe also better to call it ExportImage, as there is only one way of export, so we don't need the Tarball in the name. Or do we have avariations of export / upload image? Thinking of it -- maybe it should be DownloadImage instead of ExportImage as that more close to what it does?

Reply:
Alrighty, coming right up!
ImportFromTarballBtn -> UploadImageBtn (and Form)
ExportImageTarballBtn -> DownloadImageBtn (and Form)